src/lib.rs

```diff
@@ -72,12 +70,19 @@ use syn::ItemFn;
 /// ```
 #[proc_macro_attribute]
 pub fn test(attr: TokenStream, item: TokenStream) -> TokenStream {
+  let item = parse_macro_input!(item as ItemFn);
+  try_test(attr, item)
+    .unwrap_or_else(syn::Error::into_compile_error)
+    .into()
+}
+
+fn try_test(attr: TokenStream, input: ItemFn) -> syn::Result<Tokens> {
```

Comment:
Can this change be a separate commit? It seems orthogonal to the rest, no?

Reply:
Reconfigured the commit history^